homeassistant/components/iqvia/__init__.py

```diff
@@ -41,6 +47,13 @@
 PLATFORMS = [Platform.SENSOR]
 
 
+@dataclass
+class IQVIAData:
+    """Define an object to be stored in `hass.data`."""
+
+    coordinators: dict[str, DataUpdateCoordinator]
```

Comment:
There is no benefit in wrapping this in a data class when the coordinator is the only stored item in it?

Just store the coordinator directly in that case. Adding the dataclass here would only add another layer and consume memory for no reason.